src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService service = new UserService();
```

Comment:
`UserService` must be a class field
```suggestion
```

Reply:
Can you explain why it should be a class field?